Upload A File/Download Upload Library.gcomp/javascript/UploadFileToWebVI.js

```diff
@@ -12,46 +12,96 @@
 // http://jsfiddle.net/VBJ9h/319/
 // https://github.com/ni/webvi-examples
 // https://developer.mozilla.org/en-US/docs/Web/API/Window
-
-var fileText = "";
```

Comment:
As a side note, it's important to put all code inside of the use strict block.

Whether it is intentional or not what this code is doing because it is outside of use strict is creating the global variables window.fileText, window.fileRead, and window.inputElementName.

My guess is they were not intended to be global and placing them inside the use strict block would have made them accessible to all of your functions but not have put them in the global scope.